schema/meltano.schema.json

```diff
@@ -688,6 +722,7 @@
               "settings_group_validation": {},
               "commands": {},
               "requires": {},
+              "env": {},
```

Comment:
@WillDaSilva do we need an update to https://docs.meltano.com/reference/plugin-definition-syntax ?

Reply:
Yes. Thanks for calling that out.

Reply:
e6fef197dc1305198675682ac19cb6964a89fa88